edenia/antelope-tools
# Feat/bps list page redesign

### Redesign of BPs List

### What does this PR do?

- Apply redesign of block producers page
- Add icons for endpoints pages
- Add border radius to all buttons to keep consistency in the styles
- Resolve #1329

### Steps to test

1. Run the project locally
2. Go to Block Producers Page
3. Check the design and links to the BP Profile Page

### Screenshots

# Block Producer Profile

![imagen](https://github.com/edenia/antelope-tools/assets/66583677/77ae65d3-0476-4b10-bf1e-68ffa56a2c90)

# Block Producers List

![imagen](https://github.com/edenia/antelope-tools/assets/66583677/40f5226e-6016-4f99-852e-c4b114abcc46)
![imagen](https://github.com/edenia/antelope-tools/assets/66583677/8b1b36c9-aaad-47b9-8b01-cdb41d584277)
![imagen](https://github.com/edenia/antelope-tools/assets/66583677/cfcc1530-2617-4da8-854b-8e0ad5e6b81b)

# Network Selector

![imagen](https://github.com/edenia/antelope-tools/assets/66583677/de6d47e4-f357-4c51-bf5d-9249077a7e6c)
